docs/content/latest/api/ysql/datatypes/type_datetime/date-time-data-types-semantics/type-interval/interval-arithmetic/moment-moment-overloads-of-minus.md

```diff
@@ -52,24 +52,31 @@ This is the result:
 
 ```plpgsql
-select '2021-03-13 18:00:00'::timestamptz + '1 day 01:00:00'::interval;
+select (
+    ('2021-03-13 18:00:00 America/Los_Angeles'::timestamptz + '1 day 01:00:00'::interval) =
+    ('2021-03-14 20:00:00 America/Los_Angeles'::timestamptz)
+  )::text;
 ```
 
-This is the result:
+The result is _false_. In other words, this usual rule of arithmetic doesn't apply:
 
 ```output
-2021-03-14 19:00:00-07
+IF c ◄— a - b  THEN  a ◄— c + b
 ```
 
-You don't get back the _timestamptz_ value that you started with.
+The critical feature of this example  with respect to what seems to be a wrong result is that 18:00 on 13-Mar-2021 in the 'America/Los_Angeles' timezone is _before_ the "spring forward" moment when Daylight Savings Time starts, and that 20:00 and 18:00 on 14-Mar-2021 in that timezone is _after_ the "spring forward" moment.
+
+It's possible to understand, and predict, this outcome (and other hybrid _interval_ arithmetic outcomes like it) by understanding the three different kinds of semantics for the moment-_interval_ overloads of the `+` and `-` operators for pure _mm_ _interval_ values, pure _dd_ _interval_ values, and pure _ss_ _interval_ values.
+
+These rules are explained in the section [The moment-_interval_ overloads of the "+" and "-" operators for _timestamptz_, _timestamp_, and _time_](../moment-interval-overloads-of-plus-and-minus/). The rules for the trickiest case, moment-_interval_ arithmetic using a _pure days interval_ value, are explained in exhaustive detail in the section [Sensitivity of timestamptz-interval arithmetic to the current timezone](../../../../timezones/timezone-sensitive-operations/timestamptz-interval-day-arithmetic/).
 
-The critical feature of this example that least to what seems to be a wrong result is that 18:00 on 13-Mar-2021 in the 'America/Los_Angeles' timezone is before the "spring forward" moment when Daylight Savings Time starts, and that 20:00 and 19:00 on 14-Mar-2021 in that timezone are both after the "spring forward" moment.
+However, when you have a hybrid _dd_ and _mm_ _interval_ value, you need also to understand the priority rule: in which order are the different variations of addition/subtraction semantics done?
 
-It's possible to understand, and predict, this outcome (and other hybrid _interval_ arithmetic outcomes like it) by understanding the three different kinds of semantics for the moment-_interval_ overloads of the `+` and `-` operators for pure _mm_ _interval_ values, pure _dd_ _interval_ values, and pure _ss_ _interval_ values. But when you have a hybrid _dd_ and _mm_ _interval_ value, you need also to understand the priority rule: in which order are the different variations of addition/subtraction semantics done?
+{{< tip title="Avoid arithmetic that uses hybrid 'interval' semantics." >}}
```

Comment:
As earlier... And in general, avoid adding formatting in headings and other titles. Sometimes it's useful, but in general, less is more.
```suggestion
{{< tip title="Avoid arithmetic that uses hybrid interval semantics." >}}
```